mParticle-Adobe-Media/MPIAdobe.m

```diff
@@ -1,9 +1,14 @@
+// Be sure to apply changes to both copies of this file in the repository
```

Comment:
Is there a way we can delete one of these. Or is it possible the media and non-media one could diverge?